infra/license/src/main/java/com/simprints/infra/license/LicenseUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.simprints.infra.license
+
+import com.simprints.infra.license.remote.License
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+import java.util.TimeZone
+
+
+private fun License.isExpired(): Boolean {
+    if (expiration.isEmpty()) return false
+
+    val format = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US)
+    format.timeZone = TimeZone.getTimeZone("UTC")
+    val expiryDate = format.parse(expiration)
```

Comment:
Are we fine with the runtime exceptions being thrown if the `License::expiration` cannot be parsed? In my opinion, we should return `LicenseStatus.INVALID` in case the expiration date field (or any other field for that matter) has an invalid format.